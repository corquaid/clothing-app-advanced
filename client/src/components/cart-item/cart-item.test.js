import React from "react";
import { shallow } from "enzyme";
import { CartItem } from "./cart-item";

// single render test not requiring describe grouping

it("Should render the CartItem component", () => {
    const mockItem = {
        imageUrl: "www.testimage.com",
        price: 20,
        name: "jacket",
        quantity: 1,
    };

    expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot();
});
