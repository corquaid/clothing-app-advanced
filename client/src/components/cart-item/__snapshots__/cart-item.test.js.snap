// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the CartItem component 1`] = `ShallowWrapper {}`;
